mitmproxy/addons/proxyserver.py

```diff
@@ -31,9 +31,9 @@
 
 class Servers:
     def __init__(self, manager: ServerManager):
-        self.updating = signals.AsyncSignal(lambda old_modes, new_modes: None)
+        self.updating = signals.AsyncSignal(lambda: None)
```

Comment:
I'm a bit sad to see this go, as any listener will now have to keep a copy of the instances to get a diff, but I guess if it's not needed then it's just needless complexity.

Reply:
See above - absolutely not opposed, happy to add again if we have a use case :smiley: